src/main/scala/scalaz/stream/writer.scala

```diff
@@ -0,0 +1,20 @@
+package  scalaz.stream
+
+import scalaz.\/._
+
+
+object writer {
+
+  /** Promote a `Process` to a `Writer` that writes nothing. */
+  def lift[F[_], A](p: Process[F, A]): Writer[F, Nothing, A] =
```

Comment:
@pchiusano I will post some stuff we use for writer syntax and are not part yet in lib. If @alissapajer can perhaps just cut and past them into lib? Into WriterSyntax...